.github/workflows/transports-release.yml

```
@@ -252,3 +271,11 @@
           echo "" >> $GITHUB_STEP_SUMMARY
           echo "### Default behavior:" >> $GITHUB_STEP_SUMMARY
           echo "Changes to \`transports/go.mod\` without \`--trigger-release\` flag will not trigger a release" >> $GITHUB_STEP_SUMMARY
+
+      - name: Discord Notification - Skipped Release
+        run: |
+          MESSAGE="⏸️ **Transport Release Skipped**\n**Reason**: ${{ needs.check-release-flags.outputs.skip_reason }}\n**Branch**: \`main\`\n**Commit**: \`${{ github.sha }}\`\n**Author**: ${{ github.actor }}\n**[View Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})**"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)